deployment/terraform/assets/variables.tf

```diff
@@ -54,7 +54,7 @@ variable "db_cluster_identifier" {
 variable "db_engine_version" {
   type = map(any)
   default = {
-    "aurora-mysql"      = "8.0.mysql_aurora.3.05.0"
+    "aurora-mysql"      = "8.0.mysql_aurora.3.06.0"
```

Comment:
Can we use `3.05.2` instead? The idea was to use the same MySQL patch version as our min supported one which is `8.0.32`.

Reply:
Ah, of course, hadn't thought of that. Let me change it.